### ISSUE TITLE
(Slightly) unclear license

### ISSUE THREAD

**Issue by osfanbuff63** [booky10/StackDeobfuscator#4]
(I pretty much never delete issue templates, but this doesn't fit any of them)
What is the license of this project?

- On Modrinth, the license is listed as LGPL-3.0
- In the `LICENSE` file, both the LGPL and GPL licenses are in that file.

What's the actual license?

**Comment by booky10** [comment]
Modrinth also has the GPL-3.0 text directly under the LGPL-3.0 text.
The [lgpl page of the gnu project](https://www.gnu.org/licenses/lgpl-3.0.html) also links to two versions of the license: [plain text](https://www.gnu.org/licenses/lgpl-3.0.txt) and [LGPL+GPL](https://www.gnu.org/licenses/lgpl+gpl-3.0.txt).
I don‘t really know why, so I thought you need to have the GPL text under the LGPL text.

TL;DR it is LGPL-3.0
